xclim/_scripting.py

```diff
@@ -0,0 +1,51 @@
+import logging
```

Comment:
Is all this needed for a normal logging experience? I'm not sure why xclim would offer so many utilities and tools, it's a library not a framework?

Reply:
These have now all been removed.